File: 프로그래머스/unrated/181904. 세로 읽기/세로 읽기.java
class Solution {
    public String solution(String my_string, int m, int c) {
        StringBuilder sb = new StringBuilder( );
		for ( int i = 0; i < ( int )Math.ceil( ( double )my_string.length( ) / m ); i++ ) {
			sb.append( my_string.substring( i * m, i * m + m )
								.charAt( c - 1 ) );
		}
		return sb.toString( );
    }
}

File: 프로그래머스/unrated/181904. 세로 읽기/세로 읽기.js
function solution(my_string, m, c) {
    let result = "";
    for (let i = 0; i < my_string.length; i++) {
        if (i * m + m <= my_string.length) 
            result += my_string.substring(i * m, i * m + m).charAt(c - 1);
           
    }
    return result;
}